GoogleForCreators/web-stories-wp
# Workspace: Inherit box-sizing from design system global styles

## Context

We were developing in storybook without `border-box` sizing by default. All styles in the dashboard and editor use this sizing system, so this pr lines us up nice ✨.

## Summary

This PR does a few things:
- Updates storybook to use `box-sizing: border-box;` by default. This will make sure we develop components using the same box sizing as the dashboard and editor.
- Centralizes the box-sizing styles to one place: the design system.
- Fixes toggle styling (using border-box sizing now)
- Fixes cursor on checkbox

## Relevant Technical Choices

- Created a new global style called `Styles` which will have all the global styling in one place

## To-do

After this is merged: need to remove these changes: https://github.com/google/web-stories-wp/pull/6759/files#diff-47c08de7f287efa529a3ddc68420bfda8f7edf0b8cd8ebde09f417622ce2dadbR42-R78. We don't need those after the box-sizing is added here.

## User-facing changes

Only one change should be visible in the app: the toggle shouldn't look weird anymore.

|Before|After|
|--|--|
|![image](https://user-images.githubusercontent.com/22185279/111231582-274c1280-85af-11eb-8d04-99a43c4eb30a.png)|![image](https://user-images.githubusercontent.com/22185279/111231486-ff5caf00-85ae-11eb-8f9f-246ec4463b32.png)|
|![image](https://user-images.githubusercontent.com/22185279/111231641-3c28a600-85af-11eb-89eb-99085bc8167f.png)|![image](https://user-images.githubusercontent.com/22185279/111231621-3632c500-85af-11eb-98d1-49ac86cfc9e3.png)|



## Testing Instructions

Browse around the app, interact with form elements and buttons. Everything should look the same except the toggle. Sorry, I wish I had better testing instructions 😓.

### QA




This PR can be tested by following these steps:

1. Browse around the app, interact with form elements and buttons. Everything should look the same except the toggle. Sorry, I wish I had better testing instructions 😓. This change touches everything but nothing (except the toggle) should look different.

### UAT






## Reviews

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

no

### Does this PR have a legal-related impact?

no

## Checklist




---



Fixes #6775
